### ISSUE TITLE
Update certificate issuance doc

### ISSUE THREAD

**Issue by haydentherapper** [sigstore/fulcio#699]
**Description**

- [x] Discuss short lived certificates and how they work in the ecosystem
- [x] Update list of signing backends
- [x] Note that the CT log is separate from Rekor
- [x] Explain SCT in more detail
- [x] Update image in section 7, SCT is now embedded
- [x] Update image in section 6 for embedded flow
- [x] Proofread

**Comment by haydentherapper** [comment]
Hey @nsmith5, how did you create the images for https://github.com/sigstore/fulcio/blob/main/docs/how-certificate-issuing-works.md? Do you have editable versions of these?

**Comment by nsmith5** [comment]
Made them all with excalidraw and the source is embedded in the image so
you can open there and edit any of them

**Comment by dlorenc** [comment]
Not sure why this would block GA, the specification work is important too but separate IIUC.

**Comment by haydentherapper** [comment]
Docs that are outdated should be blockers, but low priority (P2/3) in my opinion. This is also almost done, just waiting on someone to LGTM the PRs.

**Comment by dlorenc** [comment]
> Docs that are outdated should be blockers, but low priority (P2/3) in my opinion.

This feels like an oxymoron - IMO blockers are really only the high priority issues that we MUST have. I don't think it'll come up here but in general there shouldn't be low priority blockers.
